app/controllers/participants_controller.rb

```diff
@@ -67,9 +53,6 @@ def update
     end
   end
 
-  def welcome
```

Comment:
In Rails 4+, don't need empty controller action. Declaring the route in `config/routes.rb` is sufficient.